driver/docker/docker_integration_test.go

```diff
@@ -0,0 +1,37 @@
+package docker
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/deislabs/cnab-go/driver"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDriver_Run(t *testing.T) {
+	op := &driver.Operation{
+		Installation: "example",
+		Action:       "install",
+		Image:        "pvtlmc/example-outputs@sha256:568461508c8d220742add8abd226b33534d4269868df4b3178fae1cba3818a6e",
+		Outputs:      []string{"/cnab/app/outputs/output1", "/cnab/app/outputs/output2"},
+	}
+
+	var output bytes.Buffer
+	op.Out = &output
+	op.Environment = map[string]string{
+		"CNAB_ACTION":            op.Action,
+		"CNAB_INSTALLATION_NAME": op.Installation,
+	}
+
+	docker := &Driver{}
+	docker.SetContainerOut(op.Out) // Docker driver writes container stdout to driver.containerOut.
+	opResult, err := docker.Run(op)
```

Comment:
Since this tries to connect to a Docker daemon, we should also update CI to handle this scenario:

```
--- FAIL: TestDriver_Run (0.00s)
    docker_integration_test.go:30: 
        	Error Trace:	docker_integration_test.go:30
        	Error:      	Received unexpected error:
        	            	cannot create container: Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
        	Test:       	TestDriver_Run
```

This is why CI is currently failing.

Reply:
Yeah, and this PR should probably also update the README to make it clear that you need a docker daemon in order to run the tests.